packages/block-editor/src/store/private-selectors.js

```diff
@@ -380,3 +383,47 @@ export function isDragging( state ) {
 export function getExpandedBlock( state ) {
 	return state.expandedBlock;
 }
+
+/**
+ * Retrieves the client id of the section container block.
+ *
+ * @param {Object} state Block editor state.
+ *
+ * @return {string|null} The client ID of the section container block,
+ *                       null if not found.
+ */
+export const getSectionsContainerClientId = createRegistrySelector(
```

Comment:
👍 This should stay private forever, I think this concept of "sections container" is a bit adhoc and might not be exposed to users.